doc/src/arch/reference.rst

```diff
@@ -1183,14 +1183,28 @@ The following tags are common to all ``<tile>`` tags:
 
     .. arch:tag:: <site pb_type="string"/>
 
+    Each instance of site must also specify the direct connections between the physical
+    tile pins and the logical block pins.
+
+    .. arch:tag:: <direct from="string" to="string">
+
+    Attributes:
+        - ``from`` is relative to the physical tile pins
+        - ``to`` is relative to the logical block pins
+
```

Comment:
As a sub-tag of `<equivalent_sites>` this should be indented, and probably put below the equivalent sites example.